ReportingServicesTools/Functions/CatalogItems/Update-RsSubscription.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Update-RsSubscription
+{
+  <#
+        .SYNOPSIS
+            This script will update subscriptions piped from get-RsSubscriptions
+
+        .DESCRIPTION
+            This script will take the custom object producted by get-RSSubscription and use the data to update the 
+            matchdata xml to either a new startdatetime, or enddate.
+
+        .PARAMETER ReportServerUri
+            Specify the Report Server URL to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+
+        .PARAMETER Credential
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+
+        .PARAMETER Proxy
+            Report server proxy to use.
+            Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
+            Useful when repeatedly having to connect to multiple different Report Server.
+
+        .EXAMPLE
+            $proxy = "http://ssrs-t-1234/Reportserver"

```

Comment:
for the basic example you could omit the -ReportServerUri parameter (it will be use localhost by default) , omitting  the -ReportServerUri make it more easy to follow